website/components/News.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import facepaint from 'facepaint';
+import ArrowRightIcon from '@leafygreen-ui/icon/dist/ArrowRight';
+import { css } from 'emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { Overline, Subtitle, H2 } from '@leafygreen-ui/typography';
+import { spacing, breakpoints } from '@leafygreen-ui/tokens';
+
+const mq = facepaint(
+  Object.values(breakpoints).map(bp => `@media (min-width: ${bp}px)`),
+  { literal: true },
+);
+
+const newsContainer = css`
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-bottom: ${spacing[4]}px;
+
+  ${mq({
+    marginTop: [`${spacing[4]}px`, 'unset'],
+    marginBottom: [`${spacing[4]}px`, 'unset'],
+    paddingLeft: [`${spacing[4]}px`, 'unset'],
+  })}
+`;
+
+const subtitleStyle = css`
+  cursor: pointer;
+  text-decoration: none;
+  font-weight: bolder;
+  display: inline-flex;
+  align-items: center;
+
+  &:hover {
+    & > svg {
+      opacity: 1;
+      transform: translate3d(3px, 0, 0px);
+    }
+  }
+`;
+
+const iconStyle = css`
+  transform: translate3d(-3px, 0, 0px);
+  transition: all 100ms ease-in;
+
+  ${mq({
+    opacity: [1, 1, 0, 0],
+    marginLeft: [`${spacing[2]}px`, `${spacing[2]}px`, 0],
+  })}
+`;
+
+const overlineMargin = css`
+  margin-top: 28px;
+`;
+
+const overlineColor = css`
+  color: ${uiColors.gray.dark1};
+`;
+interface UpdateProps {
+  date: string;
+  story: string;
+  path?: string;
+  href?: string;
+}
+
+function Update({ date, story, path, href }: UpdateProps) {
+  const { push } = useRouter();
+  let renderedStory: React.ReactElement | null;
+
+  if (path) {
+    renderedStory = (
+      <Subtitle as="p" className={subtitleStyle} onClick={() => push(path)}>
+        {story}
+        <ArrowRightIcon className={iconStyle} />
+      </Subtitle>
+    );
+  } else {
+    renderedStory = (
+      <Subtitle as="a" className={subtitleStyle} href={href}>
+        {story}
+        <ArrowRightIcon className={iconStyle} />
+      </Subtitle>
+    );
+  }
+
+  return (
+    <div className={overlineMargin}>
+      <Overline className={overlineColor}>{date}</Overline>
+      {renderedStory}
+    </div>
+  );
+}
+
+function News() {
+  return (
+    <div className={newsContainer}>
+      <H2 as="h1">Whats New</H2>
+      <Update
+        date="Dec 8, 2020"
```

Comment:
lmk what you think of the new approach

Reply:
A question I have is whether this approach will actually use the viewer's locale if it's SSR. Do we need to something to force the date to re-render on the client side? Also, I believe when the date is parsed, it's parsed in UTC+0, not EST, so this could lead to a date being rendered different than even we'd expect.

Reply:
As far as timeZone goes it looks like the default is: 

```timeZone
The time zone to use. The only value implementations must recognize is "UTC"; the default is the runtime's default time zone. Implementations may also recognize the time zone names of the IANA time zone database, such as "Asia/Shanghai", "Asia/Kolkata", "America/New_York".```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat